### DISCUSSION TITLE
[add] Alert History

### DISCUSSION THREAD

**Discussion by unnuetz** [grafana/grafana#43896] [Alerting]
Are there any plans about Grafana ng alerts history?

If possible, I would like to have within grafana to see history of fired alerts and revolved alerts per host.

How did you solve that, for statistics, how many alerts per week/month ...
How many alerts per Host ...

BR, Sven

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
